Expand contributing guidelines (#834)
zalando/restful-api-guidelines: CONTRIBUTING.adoc

= Contributing

== When to contribute?

=== Minor improvements

Minor improvements like grammar, spelling or link fixes are always welcome – both
from Zalando employees and from outsiders.

Just open a pull request. The maintainers will review it and usually merge.
If you see something wrong, but don't know how to fix it, please open an issue.

=== Major rule changes

For major changes to an existing rule (or adding a new rule), a discussion
and decision by Zalando's API guild is needed.

> For this, we generally only consider arguments which are relevant to Zalando's

Reviewer: ```suggestion
For this, we generally only consider arguments which also apply to Zalando's
```

situation, as these are Zalando's guidelines, not generic guidelines for everyone.

> So such changes generally will be suggested by Zalando employees.

Reviewer: ```suggestion
So such changes typically (but not exclusively) will be suggested by Zalando employees.
```

> Feel free to open an issue first outlining the need, or contact us in our

Reviewer: ```suggestion
Feel free to open an issue first outlining the need, or employees may contact us in our
```

internal #guild-api chat, or come to one of our bi-weekly meetings to discuss this.

> After we have some general alignment on what kind of rule (change) we want, someone will be assigned to write a PR. We welcome volunteers.

Reviewer: Do you want to not allow PRs for Major Rule Changes?
It did not happen in the past, and I would not forbid it. 
Add:

You may also create a PR, but we recommend to start with an Issue.

Author: I want to avoid people investing a lot of time for writing a new rule, and then being disappointed when we say that we can't accept it.

```suggestion
After we have some general alignment on what kind of rule (change) we want, someone will be assigned
to write a PR. We welcome volunteers (please note in your issue that you volunteer, if you can't be in the
meeting).

You may also directly create a PR, but to avoid spending time on big changes which won't be accepted,
we recommend to start with an issue.
```
